Nos/Extensions/Date+Elapsed.swift

```diff
@@ -69,9 +69,11 @@ extension Date {
             return formattedDate
         }
 
-        if let minute = components.minute, let formattedDate =
+        if let minute = components.minute, minute >= 1, let formattedDate =
             dateComponentsFormatter.string(from: DateComponents(calendar: calendar, minute: max(1, minute))) {
             return formattedDate
+        } else if let minute = components.minute, minute < 1 {
```

Comment:
I know we don't want to do too much nesting, but when the first statement of the conditional matches exactly with the previous one, I always think nesting makes it easier to read. I'd have a single `if let minute = components.minute`, then inside of that check for the `minute >= 1`. Something like this:

```
if let minute = components.minute {
    if minute >= 1 {
        ...
    } else {
        return String(localized: .localizable.now)
    }
}
```